XenAdmin/Wizards/PatchingWizard/PlanActions/PatchPrecheckOnHostPlanAction.cs

```diff
@@ -63,7 +63,7 @@ protected override void RunWithSession(ref Session session)
         {
             var coordinator = Helpers.GetCoordinator(Connection);
 
-            var mapping = (from HostUpdateMapping hum in mappings
+            HostUpdateMapping mapping = (from HostUpdateMapping hum in mappings
                 let xpm = hum as XenServerPatchMapping
                 where xpm != null && xpm.Matches(coordinator, xenServerPatch)
                 select xpm).FirstOrDefault();

```

Comment:
I think the way it is now it is more efficient because the casting is performed only on the found element, while if we put it in the linq expression it is performed on all mappings that were selected.